plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-product-variations-controller.php

```diff
@@ -99,7 +99,7 @@ public function prepare_object_for_response( $object, $request ) {
 			'download_limit'        => '' !== $object->get_download_limit() ? (int) $object->get_download_limit() : -1,
 			'download_expiry'       => '' !== $object->get_download_expiry() ? (int) $object->get_download_expiry() : -1,
 			'tax_status'            => $object->get_tax_status(),
-			'tax_class'             => $object->get_tax_class(),
+			'tax_class'             => $object->get_tax_class( $context ),
```

Comment:
This makes sure we get the correct `tax_class` value, this was important because it defaults to `view`, which in this case means it doesn't return `parent` if it was set to that, and instead returns the `parents` value.
This would make things quite difficult on the client side to see when the `parent` value was selected.